langstream-cli/src/test/java/ai/langstream/cli/commands/applications/UIAppCmdTest.java

```diff
@@ -24,8 +25,12 @@ class UIAppCmdTest {
     @Test
     void openBrowser() {
         // ok
-        UIAppCmd.openBrowserAtPort("echo", 9029);
+	String command = "/usr/bin/echo";
+	boolean Result = UIAppCmd.checkAndLaunch( command, 80);
```

Comment:
I think that we can remove checkAndLaunch given your findings on Desktop.browse()
at that point we can drop this test.

Reply:
Desktop.browse() didn't work in WSL linux.  Or rather it didn't when I tried it before.  That's why I went to xdg-open. 
 I can use Desktop.browse() to open the browser on a mac.  Will rewrite for mac and also try it again on WSL

Reply:
left checkAndLaunch in as WSL as Desktop.browse() fails on WSL linux (ubuntu 22.04).  Maybe it'll get fixed eventually.